src/main/java/uk/ac/ebi/ena/webin/cli/service/StudyService.java

```diff
@@ -61,28 +70,35 @@ private static class StudyResponse {
     getStudy(String studyId, String userName, String password, boolean test) {
         RestTemplate restTemplate = new RestTemplate();
 
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.